src/views/TheHome.vue

```diff
@@ -1,28 +1,35 @@
 <template>
   <section>
-    <img class='puppy-picture' alt='picture of two puppies in the snow' src='https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSrnUwJ9xC5z-HtpYQZc3jHYJfcMtQUf8qzVg&usqp=CAU'/> 
+    <img
+      class="puppy-picture"
+      alt="picture of two puppies in the snow"
+      src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSrnUwJ9xC5z-HtpYQZc3jHYJfcMtQUf8qzVg&usqp=CAU"
+    />
     <h2>Find your pup a play date at the best dog parks around!</h2>
-    <button class='button-get-started'>Get Started - woof</button>
-    <!-- onClick - take user to search component, TBD -->
+    <FindPark />
   </section>
 </template>
 
 <script>
+import FindPark from './FindPark.vue';
 
+export default {
+  components: { FindPark },
+};
 </script>
 
 <style scoped>
```

Comment:
We should make sure we are consistently using SASS in our components. I'm pretty sure I also am only using CSS in my `ResultsList` branch.

Reply:
Agreed.  I used it a bit in `ResultsItemDetails` with nesting for button styling on hover.  Using it more consistently would be good practice and keep our files consistent with syntax.